tests/test_decryption_key_index.rs

```diff
@@ -26,30 +26,32 @@ fn test_decryption_key_index_computation() {
         // the leaf, nor the common ancestor closest to the root. To do that, we
         // first have the member at index 0 remove the one at index 2, thus
         // populating its own parent node.
-        let (_, remover_id) = &group.members[0].clone();
-        let (_, target_id) = &group.members[2].clone();
+
+        // Find the identity of the member with leaf index 0.
+        let (_, remover_id) = &group
```

Comment:
This should probably a function on the test framework (`member_by_index` or something). But let's leave it to a follow up.